modules/performing-an-image-signature-verification.adoc

```diff
@@ -0,0 +1,123 @@
+. Scroll down to the *Signature Integrations* section, and click *Signature*.
+. To create a new signature integration, click *New integration*.
+. Enter a name for the integration. You can save the integration only after you specify a name for the integration.
+. Optional: To add a new public key, complete the following steps:
++
+[NOTE]
+====
+** If you add a public key, you do not need to create a new certificate verification.
+** You can add one or more public keys.
+** You can save the integration only after you specify a name and value for the public key.
+====
+.. Expand *Cosign public Keys*, and then click *Add new public key*.
+.. Enter a name for the key. 
+.. Enter a value for the key which is encoded in PEM format.
+
+. Optional: To add a new certificate verification, complete the following steps:
++
+[IMPORTANT]
+====
+** When you create a signature integration, you must add a new certificate verification if you want to use keyless verification for the image signature by using {rh-rhtas-first}.
+** You can add one or more certificate verifications.
+** You can save the integration only after you specify a certificate OpenID Connect (OIDC) issuer and certificate identity for the certificate verification.
+====
+.. Expand *Cosign certificates*, and then click *Add new certificate verification*.
+.. Enter the certificate OIDC issuer that Cosign specifies. You must use regular expressions in RE2 syntax for matching.
++
+For more information, go to the GitHub repository at `google/re2`, open the `Wiki` section, and then select the `Syntax` page.
+
+.. Enter the certificate identity that Cosign specifies. You must use regular expressions in RE2 syntax for matching.
++
+For more information, go to the GitHub repository at `google/re2`, open the `Wiki` section, and then select the `Syntax` page.
+.. Enter the trusted certificate root which is encoded in PEM format to verify the certificates. If you do not specify the certificate root, the public Fulcio roots are used automatically for verification. 
++
+For more information, see link:https://docs.sigstore.dev/certificate_authority/overview/[Fulcio] (Sigstore documentation).
+
+.. Enter the trusted signer intermediate certificate authority to verify the certificates. If you do not specify the certificate authority, the certificate chain is used automatically for verification.
+.. Optional: Select the *Enable certificate transparency log validation* checkbox to validate the proof of inclusion into the certificate transparency log.
++
+Enter the public key that you want to use to validate the proof of inclusion into the certificate transparency log.
+If you do not specify the public key, the key of the public Sigstore instance is used automatically for validation.
+
+. Optional: To configure the transparency log, complete the following steps:
++
+[NOTE]
+====
+When you create a signature integration, you can enable the validation of transparency logs in the following situations:
```

Comment:
We're back to signature verification? All the more reason that this should be a separate module.